instaloader/instaloader.py

```diff
@@ -527,7 +555,8 @@ def download_post(self, post: Post, target: Union[str, Path]) -> bool:
         downloaded = True
         if post.typename == 'GraphSidecar':
             if self.download_pictures or self.download_videos:
-                for edge_number, sidecar_node in enumerate(post.get_sidecar_nodes(), start=1):
+                for edge_number, sidecar_node in enumerate(post.get_sidecar_nodes(self.slide_start,
+                                                                                  self.slide_end), start=1):
```

Comment:
```suggestion
                for edge_number, sidecar_node in enumerate(post.get_sidecar_nodes(self.slide_start,
                                                                                  self.slide_end), start=self.slide_start+1):
```

because otherwise the filenames are wrong. (`--slide last` currently names all files `..._1.jpg`).